static/js/pm_list.js

```diff
@@ -76,35 +133,143 @@ export function update_dom_with_unread_counts(counts) {
     set_count(counts.private_message_count);
 }
 
-function should_expand_pm_list(filter) {
-    const op_is = filter.operands("is");
+export function update_collapsed_private_messages_section() {
+    // when we need to update the collapsed private messages section, we first check
```

Comment:
Also this function doesn't need to be exported.

Reply:
Hmm, sorry for this kind of mistake, would try to not let this happen again, thanks!
Updated the PR.

Reply:
I wish our linter found things like that! That would be very helpful